src/com/carolinarollergirls/scoreboard/core/impl/ScoreBoardImpl.java

```diff
@@ -46,6 +47,9 @@ public ScoreBoardImpl() {
     }
 
     protected void setupScoreBoard() {
+        if (getCurrentTimeout() == null) {
+          set(Value.CURRENT_TIMEOUT, noTimeoutDummy);
+        }
```

Comment:
What is this doing here?

Reply:
Preventing an NPE, but I handled it in another way.